lib/liquid_voting/voting.ex

```diff
@@ -320,7 +362,9 @@ defmodule LiquidVoting.Voting do
       {:error, %Ecto.Changeset{}}
 
   """
-  def delete_participant(%Participant{} = participant), do: Repo.delete(participant)
+  def delete_participant(%Participant{} = participant) do
```

Comment:
better form to keep it a one-liner

Reply:
ah yes, this was a hangover from inserting debugging lines. thanks